### PR TITLE
Fix/use permitted params for board

### PR DESCRIPTION
**Pull request by tessi**
see: https://www.openproject.org/work_packages/3947

### CODE REVIEW

File: app/models/permitted_params.rb [hunk 1]
```diff
@@ -249,6 +257,11 @@ def permitted_attributes(key, additions = {})
 
   def self.permitted_attributes
     @whitelisted_params ||= {
+<<<<<<< HEAD
```
Comment (ulferts) [inline app/models/permitted_params.rb]: Merge left overs

Comment (tessi) [inline app/models/permitted_params.rb]: ewwww

**Comment by ulferts** [comment]
This failed :(

**Comment by linki** [comment]
rebased and hopefully fixed the tests. fingers crossed...

**Comment by tessi** [comment]
probably the tests still fail (they need to be changed :)) - forgot to do that on monday

**Comment by linki** [comment]
changed them :) but others are failing

**Comment by linki** [comment]
green!

**Comment by mfrister** [comment]
:+1:

**Pull request merged**